### ISSUE TITLE
[Bug]: original audio stream is replaced by AI generated spannish overdub

### ISSUE THREAD

**Issue by nekkz** [FreeTubeApp/FreeTube#6353]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1. Click a video, by example that one, https://youtu.be/gP5xc8OEka8
2. Ensure you are in Dash format
3. Play
4. The video starts with some sort of AI generated overdub in spanish

### Expected Behavior

It started to happen this week, probably Youtube are trying some sort of automatic overdub generation and FreeTube is taking the first audio stream available.

I never asked for an automatic translation, didn't change settings in the last month, I'd expect the audio stream to be the original. My Freetube localization is English and I live in Canada.

### Issue Labels

feature stopped working

### FreeTube Version

v0.22.0 beta

### Operating System Version

Windows 11

### Installation Method

.exe

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.22.0 bera

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by nekkz** [comment]
I'm sorry I spelled Spannish instead of spanish, so I didn't find the closed bug about it. Apparently it has already been fixed. Thank you!

**Comment by LinuxOnTheDesktop** [comment]
>  [. .] I didn't find the closed bug about it. Apparently it has already been fixed.

That report is, I think, [this one](https://github.com/FreeTubeApp/FreeTube/issues/6311).

There is further information [here](https://github.com/FreeTubeApp/FreeTube/issues/4291#issuecomment-2452527942).

And, happily, there is a release with a fix - [here](https://github.com/FreeTubeApp/FreeTube/releases).

**Comment by LinuxOnTheDesktop** [comment]
My previous comment referred (until I edited that comment), erroneously, to a release containing a fix. There is no such release, _unless_ one counts _the nightly builds_ which are, I discover, [here](https://github.com/FreeTubeApp/FreeTube/actions?query=workflow%3ABuild). 

Here is how to find the correct nightly build for one's needs and how to install it.

One has to hunt through the continuations (page 2, page 3, etc.) of the aforementioned page, so as to find the version in the language that one wants. Having click that one, one must scroll down the resulting page to find installable packages. That said, the package I found - a .deb - was zipped, so I had to unzip it first. I found also that I had to uninstall my previous build.
